Description: Use Debian specific path to perl.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/examples/perl/helloplacemark.pl
+++ b/examples/perl/helloplacemark.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # Copyright 2008, Google Inc. All rights reserved.
 #
--- a/examples/perl/hellowalk.pl
+++ b/examples/perl/hellowalk.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # Copyright 2008, Google Inc. All rights reserved.
 #
